doc/tutorials/Multi-PC.md

```diff
@@ -1,8 +1,10 @@
-# How to Run with Multiple Computers
+# Multiple PCs
 
-This walks through an approach of connecting multiple computers each running their own docker containers using host networks. There are a few other options with their own upsides/downsides that will be discussed in the Wiki.
+This walks through an approach of connecting multiple computers each running their own docker containers using host networks. There are a few other options with their own pro/cons found under [Discussion](#discussion).
 
-## Running (QTRobot)
+## Host Network Approach
+
+_This provided example covers the two PC setup that exists on QTrobot, which was an early deployment platform._
 
 Run the following commands. A heading indicating #QTRP or #QTPC will be included each time context changes. Any host or container can be ROS Master as long as the ROS_MASTER_URI is set correctly.
 
```

Comment:
I think the approaches below might need further explanation. This is actually a super important section for most use cases, and so we want to make it really clear. One thing that might not be immediately clear is what needs to be set where, and what needs to be run on which machine.

Reply:
Hmm. Maybe I should add a diagram and have clearer separations somehow? Suggestions are welcome. I worked on this with @micolspitale93 a while ago, but should probably verify how she's working with it right now.

Reply:
For Misty, we have created a docker compose which relied on the host network (as for the QT). For QT, I remember I have followed the steps of the docker run mentioned in the MULTI_PC redme.